www/src/components/forum/TopicCarrousel.tsx

```diff
@@ -0,0 +1,60 @@
+"use client";
+import type { RelatedTopic } from "@/app/forum/topic/[topicId]/page";
+import { cn } from "@/lib/utils";
+import Autoplay from "embla-carousel-autoplay";
+import React from "react";
+import {
+  Carousel,
+  CarouselContent,
+  CarouselItem,
+  CarouselNext,
+  CarouselPrevious,
+} from "./Carousel";
+import { type Category, Topic } from "./Topic";
+
+export interface TopicCarouselProps {
+  relatedTopics: RelatedTopic[];
+  category: Category;
+}
+
+export function RelatedTopicCarousel({
+  relatedTopics,
+  category,
+}: TopicCarouselProps) {
+  const plugin = React.useRef(
+    Autoplay({ delay: 3000, stopOnInteraction: true }),
+  );
```

Comment:
👀 cool, had never seen this! do we really want things moving on the screen while users are reading that content though?

Reply:
🤔 I don't think so, actually I was considering to use the autoplay only in the mobile version, but I think it won't be necessary either

Reply:
<img width="516" alt="image" src="https://github.com/user-attachments/assets/c85e6b06-03aa-4497-b8d0-4ee5326b81da">
It's already clear on mobile versions that it can be scrollable too